kolibri/plugins/device_management/assets/src/views/UserPermissionsPage/index.vue

```diff
@@ -221,12 +270,36 @@
     margin-left: 0;
   }
 
+  table {
+    width: 100%;
+    text-align: left;
+    table-layout: fixed;
+  }
+  th {
+    width: 112px;
+  }
+
+  .super-admin-checkbox {
+    display: inline-table;
+  }
+
+  .checkbox-description {
+    // visual estimate, supposed to line up with checkbox label
+    margin: 0;
```

Comment:
Maybe this instead
```css
margin: 0px 0 0 50px;
padding: 0;
```

Reply:
Is the benefit here that we're explicitly defining padding?